BGL/include/CGAL/boost/graph/named_params_helper.h

```diff
@@ -102,111 +110,255 @@ namespace CGAL {
     typedef typename boost::property_traits<PMap>::value_type type;
   };
 
-  template<typename PolygonMesh, typename NamedParameters>
-  class GetVertexPointMap
+  namespace Polygon_mesh_processing
   {
-    typedef typename property_map_selector<PolygonMesh, boost::vertex_point_t>::const_type
-    DefaultVPMap_const;
-    typedef typename property_map_selector<PolygonMesh, boost::vertex_point_t>::type
-    DefaultVPMap;
-  public:
-    typedef typename boost::lookup_named_param_def<
-    internal_np::vertex_point_t,
-    NamedParameters,
-    DefaultVPMap
-    > ::type  type;
-    typedef typename boost::lookup_named_param_def<
+  
+    template<typename PolygonMesh, typename NamedParameters>
```

Comment:
no need to change the indentation when you add a namespace (in addition it creates conflicts when merging other branches)